apps/challenges/views.py

```diff
@@ -69,6 +69,8 @@
     get_participant_team_of_user_for_a_challenge,
 )
 
+from jobs.filters import SubmissionFilter
```

Comment:
Please add this import alphabetically with other imports from `jobs` app. i.e. line 59